### DISCUSSION TITLE
Error 185e9619624fc000 / ECONNABORTED

### DISCUSSION THREAD

**Discussion by codea83** [aarons22/homebridge-bond#164] [Q&A] [answered]
Hello! My two Bonds were working fine, then one day I started getting the below errors. I've reboot all the devices including the router/and switches with no luck. I tried adding it to it's own child bridge without any changes. I am able to navigate to one of the addresses that it appears to be looking up without any issues on my phone, the other doesn't work. (http://192.168.1.231/v2/sys/version works, http://192.168.1.231/v2/devices doesn't work). Any ideas? Let me know if you need anything additional. 

Log with syntax highlighting - https://pastebin.com/depiRwej

**Comment by chrismerck** [comment, marked as answer]
From your log, it seems that HB is OK communicating with both endpoints on the IP ending 231, but on the other one ending 229 is where the timeout problem is. Maybe it's got a different IP, or it's offline, or other network problem. You can check the IP address in the Bond Home app.

> I am able to navigate to one of the addresses that it appears to be looking up without any issues on my phone, the other doesn't work. (http://192.168.1.231/v2/sys/version works, http://192.168.1.231/v2/devices doesn't work).

`sys/version` doesn't require any authentication.

`devices` endpoint requires token authentication, so that's why it's not loading on your phone. If you used a command-line like cURL to make the request, you'd see the 401 Unauthorized response.

**Reply by codea83** [reply]
Thanks so much for the quick reply, it was 100% a network issue. I had originally set some static IPs that ended up conflicting with these devices. I updated the config on them and everything is working perfectly again. Thanks again!
